internal/kafka/command_link_list.go

```diff
@@ -14,9 +14,9 @@ const includeTopicsFlagName = "include-topics"
 type listOut struct {
 	Name                 string `human:"Name" serialized:"link_name"`
 	TopicName            string `human:"Topic Name" serialized:"topic_name"`
-	SourceClusterId      string `human:"Source Cluster" serialized:"source_cluster_id"`
-	DestinationClusterId string `human:"Destination Cluster" serialized:"destination_cluster_id"`
-	RemoteClusterId      string `human:"Remote Cluster" serialized:"remote_cluster_id"`
+	SourceClusterId      string `human:"Source Cluster" serialized:"source_cluster"`
+	DestinationClusterId string `human:"Destination Cluster" serialized:"destination_cluster"`
+	RemoteClusterId      string `human:"Remote Cluster" serialized:"remote_cluster"`
```

Comment:
Should we make these `omitempty`?

Reply:
I'm not sure, it wasn't like that earlier. How do we decide that?

Reply:
Hmm. Actually maybe we should.

Even though it's possible for all three to be non-empty, usually at least one of them is. So omitempty could make it less cluttered.

Reply:
okay, these should be omitempty in `command_link_describe` as well right?

Reply:
Oh, they don't use the same output struct...

Yeah I think it should be omitempty in describe as well. Going even further, maybe we should add omitempty to `Error` and `ErrorMessage` in `describe` and unify these two structs?

cc: @brianstrauch

Reply:
i have unified both the structs